Modules/IO/NIFTI/include/itkNiftiImageIO.h

```diff
@@ -71,6 +71,19 @@ class ITKIONIFTI_EXPORT NiftiImageIO : public ImageIOBase
     OtherOrError = -1,
   };
 
+  /** Way to treat legacy Analyze75 files */
+  enum Analyze75Flavour
```

Comment:
We are in the process of switching to `enum class` style, so new code should be using that.

Reply:
enum class doesn't seem to work with itkSetMacro

Reply:
Matt just made a big commit in enum class PR, see this comment:
https://github.com/InsightSoftwareConsortium/ITK/pull/1471#issuecomment-573191946